### PR TITLE
refactor: rename send to transfer.

### PR DESCRIPTION
**Pull request by yenkhoon**
## Description
Rename send to transfer.

### CODE REVIEW

File: cmd/wallet/main.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func main() {
 	})
 	app.Command("tx", "Create, sign and publish a transaction", func(k *cli.Cmd) {
 		k.Command("bond", "Create, sign and publish a bond transaction", BondTx())
-		k.Command("send", "Create, sign and publish a send transaction", SendTx())
+		k.Command("send", "Create, sign and publish a send transaction", TransferTx())
```
Comment (b00f) [inline cmd/wallet/main.go:77]:
```suggestion
		k.Command("transfer", "Create, sign and publish a Transfer transaction", TransferTx())
```

Comment (b00f) [inline cmd/wallet/main.go:77]:
```suggestion
		k.Command("transfer", "Create, sign and publish a Transfer transaction", TransferTx())
```

File: cmd/wallet/main.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func main() {
 	})
 	app.Command("tx", "Create, sign and publish a transaction", func(k *cli.Cmd) {
 		k.Command("bond", "Create, sign and publish a bond transaction", BondTx())
```
Comment (b00f) [inline cmd/wallet/main.go:76]:
```suggestion
		k.Command("bond", "Create, sign and publish a Bond transaction", BondTx())
```

File: cmd/wallet/main.go [hunk 3]
```diff
@@ -74,7 +74,7 @@ func main() {
...
-		k.Command("send", "Create, sign and publish a send transaction", SendTx())
+		k.Command("send", "Create, sign and publish a send transaction", TransferTx())
 		k.Command("unbond", "Create, sign and publish an unbond transaction", UnbondTx())
```
Comment (b00f) [inline cmd/wallet/main.go:78]:
```suggestion
		k.Command("unbond", "Create, sign and publish an Unbond transaction", UnbondTx())
```

File: cmd/wallet/main.go [hunk 4]
```diff
@@ -74,7 +74,7 @@ func main() {
...
 		k.Command("withdraw", "Create, sign and publish a withdraw transaction", WithdrawTx())
```
Comment (b00f) [inline cmd/wallet/main.go:79]:
```suggestion
		k.Command("withdraw", "Create, sign and publish a Withdraw transaction", WithdrawTx())
```

**Review by b00f — CHANGES_REQUESTED** [review]
rename `NewSendTx` to `NewTransferTx`

File: types/tx/payload/payload.go [hunk 5]
```diff
@@ -19,7 +19,7 @@ const (
 
 func (t Type) String() string {
 	switch t {
-	case PayloadTypeSend:
+	case PayloadTypeTransfer:
 		return "send"
```
Comment (b00f) [inline types/tx/payload/payload.go:23]:
```suggestion
		return "transfer"
```

Comment (b00f) [inline types/tx/payload/payload.go:23]:
```suggestion
		return "transfer"
```

**Comment by b00f** [comment]
@yenkhoon You forgot to rename  `NewSendTx` to `NewTransferTx`

**Comment by yenkhoon** [comment]
> 

Okie i will change it

**Review by b00f — APPROVED** [review]

**Comment by b00f** [comment]
@yenkhoon Thanks

**Pull request merged**